src/style.css

```
@@ -0,0 +1,65 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.image {
+  height: 40px;
+  width: 40px;
+  align-self: center;
+}
+
+.nav-list {
+  padding: 0;
+  display: flex;
+  margin-left: -20px;
+  list-style: none;
+}
+
+.nav-item {
+  margin-left: 20px;
+}
+
+.nav-link {
+  font-weight: 500;
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  white-space: nowrap;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  position: relative;
+}
+
+.nav-link:hover {
+  color: #00acdc;
+}
+
+.nav-link.is-active {
```

Уникай такої вкладеності
```suggestion
.is-active {
```